### PR TITLE
PCHR-2704: Payroll and Emergency Contact pages styles

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR, the Payroll and the Emergency Contact Page has been styled.

## Screenshots
**Payroll Page**
localhost-8907-onboarding-form

**Emergency Contact Page**
localhost-8907-onboarding-form 1

### CODE REVIEW

File: civihr_default_theme/assets/js/civihr_default_theme.script.js [hunk 1]
```diff
@@ -3,53 +3,71 @@
...
+  };
 
   // Create theme related functions
   Drupal.civihr_theme = Drupal.civihr_theme || {};
 
-  Drupal.civihr_theme.initMobileNav = function() {
-    $header = $('.chr_header');
-    $mobileLogo = $('.chr_header__brand.chr_brand.chr_header__home-menu');
-    $nav = $header.find('.chr_header__nav');
+  Drupal.civihr_theme.onBoardingWizard = function () {
```
Comment (reneolivo) [inline civihr_default_theme/assets/js/civihr_default_theme.script.js]: I know this whole file is missing function comments, but can you at least add them to the functions you have modified or created?

File: civihr_default_theme/scss/pages/_onboarding-wizard.scss [hunk 2]
```diff
@@ -1,4 +1,14 @@
 .onboarding-wizard-page {
+  #edit-submitted-add-another-emergency-contact {
+    .form-type-radio {
+      width: 50%;
+    }
+  }
+
+  label[for="edit-submitted-civicrm-1-contact-1-cg16-custom-73"] {
```
Comment (reneolivo) [inline civihr_default_theme/scss/pages/_onboarding-wizard.scss]: is this a reliable selector? It seems auto-generated. What if it changes?

File: civihr_default_theme/scss/pages/_onboarding-wizard.scss [hunk 3]
```diff
@@ -33,11 +43,50 @@
...
   }
 
+  .form-radios {
+    background-color: transparent !important;
+
+    > div {
+      background-color: transparent !important;
+      float: left;
+      padding: 8px 12px;
+      border: 1px solid #949caa;
```
Comment (reneolivo) [inline civihr_default_theme/scss/pages/_onboarding-wizard.scss]: `border: 1px solid $crm-color-neutral`

File: civihr_default_theme/scss/pages/_onboarding-wizard.scss [hunk 4]
```diff
@@ -33,11 +43,50 @@
...
+      border-right: none;
+      margin: 0 !important;
+    }
+
+    > div:last-child {
+      border-right: 1px solid #949caa;
```
Comment (reneolivo) [inline civihr_default_theme/scss/pages/_onboarding-wizard.scss]: use `$crm-color-neutral`

**Review by reneolivo — CHANGES_REQUESTED** [review]

**Review by reneolivo — CHANGES_REQUESTED** [review]
I see there are a lot of `!important` rules. If those could be avoided it would be ideal, even though I know it can't in most cases.

File: civihr_default_theme/scss/pages/_onboarding-wizard.scss (hunk 2 above)
Comment (deb1990) [reply civihr_default_theme/scss/pages/_onboarding-wizard.scss]: Fixed

**Comment by deb1990** [comment]
> I see there are a lot of !important rules. If those could be avoided it would be ideal, even though I know it can't in most cases.

@reneolivo I have applied `!important` only if it is necessary. I have checked again, and all of them are required.

**Review by reneolivo — APPROVED** [review]

**Pull request merged**